Docs: Add Query history search documentation (#25946)
grafana/grafana: docs/sources/features/explore/index.md

> Note: If you are in split mode, filters are applied only to your currently active panel.

### Search in query history

> You can search in your history across queries and your comments. Search is possible in Query history tab and Starred tab.

Author: ```suggestion
You can search in your history across queries and your comments. Search is possible for queries in Query history tab and Starred tab.
```

> You can search in your history across queries and your comments. Search is possible for queries in Query history tab and Starred tab.

Reviewer: ```suggestion
You can search in your history across queries and your comments. Search is possible for queries in the Query history tab and Starred tab.
```

1. Click the **Search queries** field.

> 2. Type the term you are searching for into search field.

Reviewer: ```suggestion
1. Type the term you are searching for into search field.
```

Reviewer: Per the [Markdown style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.md), we want to use lazy numbering by default.

> 1. Type the term you are searching for into search field.

Author: ```suggestion
2. Type the term you are searching for into search field.
```

| ------------------------------------------------------------- | --------------------------------------- |
| Period of time for which Grafana will save your query history | 1 week                                  |
| Change the default active tab                                 | Query history tab                       |

> | Only show queries for datasource currently active in Explore  | True                                    |

Reviewer: ```suggestion
| Only show queries for data source currently active in Explore  | True                                    |
```

{{< docs-imagebox img="/img/docs/v70/explore-trace-view-timeline.png" class="docs-image--no-shadow" caption="Screenshot of the trace view timeline" >}}

Shows list of spans within the trace. Each span row consists of these components:

- Expand children button: Expands or collapses all the children spans of selected span.
- Service name: Name of the service logged the span.
- Operation name: Name of the operation that this span represents.
- Span duration bar: Visual representation of the operation duration within the trace.

> Clicking anywhere on the span row will show span details.

Reviewer: ```suggestion
Clicking anywhere on the span row shows span details.
```
